server/txSender/txSender.go

```diff
@@ -122,9 +122,10 @@ func (ts *txSender) createAndSendTxs(ctx context.Context, data *sovereign.Bridge
 			}
 		default:
 			log.Error("invalid tx data received", "data", string(txData))
+			continue
```

Comment:
I would say that this is a fatal error and should not continue...
Not very sure about it, though...

Reply:
It is better, but I meant we should directly return error, not continue.
This is fatal, we should not skip and send "half txs"